ERROR: Z3 error Sorts (Array (_ BitVec 64) struct_type_pointer_struct) and (Array (_ BitVec 32) struct_type_pointer_struct) are incompatible encountered

$esbmc argv.c --unwind 1 --z3 -Wno-error=implicit-function-declaration

````C
void main(a, argv) int a;
char **argv;
{
  for (;;)
    if (argv[1])
      break;
}
````

````
ESBMC version 7.8.1 64-bit x86_64 linux
Target: 64-bit little-endian x86_64-unknown-linux with esbmclibc
Parsing argv.c
argv.c:1:1: warning: return type of 'main' is not 'int' [-Wmain-return-type]
void main(a, argv) int a;
^
argv.c:1:1: note: change return type to 'int'
void main(a, argv) int a;
^~~~
int
argv.c:1:6: warning: a function definition without a prototype is deprecated in all versions of C and is not supported in C2x [-Wdeprecated-non-prototype]
void main(a, argv) int a;
     ^
Converting
Generating GOTO Program
GOTO program creation time: 0.423s
GOTO program processing time: 0.002s
Starting Bounded Model Checking
Not unwinding loop 2 iteration 1   file argv.c line 4 column 3 function main
Symex completed in: 0.005s (20 assignments)
Slicing time: 0.000s (removed 12 assignments)
Generated 2 VCC(s), 2 remaining after simplification (8 assignments)
Encoding remaining VCC(s) using bit-vector/floating-point arithmetic
ERROR: Z3 error Sorts (Array (_ BitVec 64) struct_type_pointer_struct) and (Array (_ BitVec 32) struct_type_pointer_struct) are incompatible encountered
Aborted (core dumped)
````

--- a/regression/esbmc/github_2377_2/main.c
+++ b/regression/esbmc/github_2377_2/main.c
@@ -1,0 +1,7 @@
+void main(a, argv) int a;
+char **argv;
+{
+  for (;;)
+    if (argv[1])
+      break;
+}

--- a/regression/esbmc/github_2377_2/test.desc
+++ b/regression/esbmc/github_2377_2/test.desc
@@ -1,0 +1,4 @@
+CORE
+main.c
+--unwind 1 --z3 -Wno-error=implicit-function-declaration --no-slice
+^VERIFICATION FAILED$

